website/docs/docs/cloud/git/setup-azure-service-principal.md

```diff
@@ -85,18 +83,6 @@ An Entra ID admin needs to provide your new app access to Azure DevOps:
 4. Select **Azure DevOps**.
 5. Select the **user_impersonation** permission. This is the only permission available for Azure DevOps.
 
-## Add a role to service principal
```

Comment:
hey @matthewshaver , [based on this thread](https://dbt-labs.slack.com/archives/C07JY95CJ2X/p1739542662554479?thread_ts=1739526418.511669&cid=C07JY95CJ2X), should this step be removed?

Reply:
Yes, we'll still remove this and lean on the MS docs. That conversation is about a separate section. Will open a separate PR for that